### PR TITLE
fix uploadprogressbar tooltip

### PR DESCRIPTION
**Pull request by burgha**
Signed-off-by: Hannes Burger <hburger30@gmail.com>

### CODE REVIEW

File: apps/files/js/operationprogressbar.js [hunk 1]
```diff
@@ -65,7 +65,7 @@
 			$('#uploadprogressbar>em').addClass('outer');
 			$('#uploadprogressbar').tooltip({placement: 'bottom'});
 			if(title) {
-				$('#uploadprogressbar').attr('original-title', title);
+				$('#uploadprogressbar').attr('data-original-title', title);
```
Comment (juliushaertl) [inline apps/files/js/operationprogressbar.js]: The issue is that the bootstrap tooltip used here doesn't support updating the content while showing, so you will need to work around that by manually setting the text of the related tooltip element. Something like that should work:

```
$('#uploadprogressbar').tooltip({placement: 'bottom', container: '#uploadprogresswrapper'});
if (title) {
	$('#uploadprogressbar').attr('data-original-title', title);
	$('#uploadprogresswrapper .tooltip-inner').text(title);
}
```

Comment (burgha) [reply apps/files/js/operationprogressbar.js]: thank you, i will give it a shot later

Comment (burgha) [reply apps/files/js/operationprogressbar.js]: your idea works flawlessly, thank you very much for helping me

**Comment by burgha** [comment]
@juliushaertl is there anything else i should change or add to this? if not i would be finished on my part

**Comment by kesselb** [comment]
Peek 2019-11-05 21-10

Works for me :+1: Sometimes the value is negative. Guess thats a hicup on localhost :confused:

**Review by kesselb — APPROVED** [review]

**Review by juliusknorr — APPROVED** [review]

**Comment by juliushaertl** [comment]
Failures unrelated:

```
PHP Fatal error: Uncaught PDOException: SQLSTATE[HY000]: General error: 2006 MySQL server has gone away in /drone/src/3rdparty/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOConnection.php:41
24
```

**Pull request merged**